eslint.config.mjs

```diff
@@ -8,5 +8,6 @@ export default antfu(
   },
   {
     ...playwright.configs['flat/recommended'],
+    ignores: ['tests/unit-tests/**/*.ts'],
```

Comment:
I'm aware this is causing a build failure. Let me know how to go about this if you have any advice. Currently looking into a solution.

Reply:
![image](https://github.com/user-attachments/assets/a9404f0c-a98b-4dc9-823e-a79e73806e73)

This property doesn't exist for [eslint-plugin-playwright](https://github.com/playwright-community/eslint-plugin-playwright).

Were the playwright ESLint rules causing issues with Vitest tests?

Reply:
<img width="1006" alt="Screenshot 2024-08-12 at 6 47 59 PM" src="https://github.com/user-attachments/assets/177592c4-f5ec-4b2c-b578-d70129256256">
ESLint misidentifies the Vitest code as playwright

Reply:
Yep working on fixing it currently

Reply:
Couldn't fix it, commented it out altogether for now. The E2E tests will be in a separate directory in the monorepo so we won't have conflicting configs anyways. Monorepo management tools handle a lot of these config headaches for you.